apps/wallet/src/utils/transaction/gasilon.ts

```diff
@@ -0,0 +1,32 @@
+import { logger, Networks } from '@walless/core';
+import { environment } from 'utils/config';
+
+export const gasilonSupportedNetworks: Networks[] = [Networks.solana];
+
+type GasilonToken = {
+	name: string;
+	mint: string;
+	decimals: string;
+};
+
+type GasilonConfig = {
+	feePayer: string;
+	transfer: {
+		tokens: GasilonToken[];
+	};
+};
+
+let gasilonConfig: GasilonConfig;
+export const getGasilonConfig = async () => {
+	if (!gasilonConfig) {
+		try {
+			const res = await fetch(environment.GASILON_ENDPOINT, { method: 'GET' });
```

Comment:
by default this should be 'GET' already, unless we want to override it to e.g POST or PUT..